packages/amplify-e2e-tests/src/__tests__/auth_2.test.ts

```diff
@@ -120,10 +122,18 @@ describe('amplify add auth...', () => {
     const functionName = `${authKey}PostConfirmation-integtest`;
     const authMeta = meta.auth[authKey];
     const id = authMeta.output.UserPoolId;
-    const userPool = await getUserPool(id, meta.providers.awscloudformation.Region);
+    const region = meta.providers.awscloudformation.Region;
+    const userPool = await getUserPool(id, region);
     const clientIds = [authMeta.output.AppClientIDWeb, authMeta.output.AppClientID];
-    const clients = await getUserPoolClients(id, clientIds, meta.providers.awscloudformation.Region);
-    const lambdaFunction = await getLambdaFunction(functionName, meta.providers.awscloudformation.Region);
+    const clients = await getUserPoolClients(id, clientIds, region);
+    await addUserToUserPool(id, region);
+    const lambdaFunction = await getLambdaFunction(functionName, region);
```

Comment:
Are we invoking the PostConfirmation trigger lambda directly instead of testing it through Cognito to see if it is properly configured and working as expected? If that's the case, we need to test it end-to-end, not in isolation like this.

Reply:
I tried with ```AdminConfirmSingup``` Call which calls PostConfirmation Auth Trigger , But there is no way currently as When I tried invoking though USerPool, got following errors : 

AdminCreateUser -> Admin ConfirmSignUp -> error: user state is FORCE_PASSWD_CHANGE(NOT Authorized)
AdminCreateUser -> AdminSetPassword ->  Admin ConfirmSignUp -> error: user state is CONFIRMED(NOT Authorized)

Reply:
We are creating and confirming users in the graphql e2e tests by using AmplifyJS calls, no need to admin confirm it, use the same method that client applications will use.